modules/docs/docgen/plugins/enhancedComponentParser.ts

```diff
@@ -471,10 +471,12 @@ export const enhancedComponentParser = createParserPlugin<SupportedValues>((node
       const type = node.initializer.typeArguments
         ? parser.checker.getTypeAtLocation(node.initializer.typeArguments[0])
         : undefined;
-      const props = (type ? getComponentProps(parser, signature, type, baseElement) : []).filter(
-        // Filter out `model` and `elemPropsHook` that might come if we extend an interface
-        p => !['model', 'elemPropsHook'].includes(p.name)
-      );
+      const props = (type ? getComponentProps(parser, signature, type, baseElement) : [])
+        .filter(
+          // Filter out `model` and `elemPropsHook` that might come if we extend an interface
+          p => !['model', 'elemPropsHook'].includes(p.name)
```

Comment:
When we filter out model, i assume that account for any state and events?

Reply:
The `model` is added as a external prop by an enhanced component. When we use `ExtractProps` of an enhanced component, the `model` and `elemPropsHook` are added. We filter those out because we add more specific ones because of the `modelHook`